geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/kafka/commands/ListCommand.scala

```diff
@@ -0,0 +1,80 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.tools.kafka.commands
+
+import com.beust.jcommander.{JCommander, Parameters}
+import com.typesafe.scalalogging.LazyLogging
+import kafka.utils.{ZKStringSerializer, ZkUtils}
+import org.I0Itec.zkclient.ZkClient
+import org.I0Itec.zkclient.exception.ZkNoNodeException
+import org.locationtech.geomesa.tools.common.commands.Command
+import org.locationtech.geomesa.tools.kafka.commands.ListCommand._
+import org.locationtech.geomesa.tools.kafka.{DataStoreHelper, OptionalZkPathParams}
+
+class ListCommand(parent: JCommander) extends Command(parent) with LazyLogging {
+  override val command = "list"
+  override val params = new ListParameters()
+  lazy val zkClient = new ZkClient(params.zookeepers, Int.MaxValue, Int.MaxValue, ZKStringSerializer)
+
+  override def execute() = {
+    if (params.zkPath == null) {
+      logger.info(s"Running List Features without zkPath...")
+      logger.info(s"zkPath - schema")
+      ZkUtils.getAllTopics(zkClient).filter(_.contains('-')).foreach {
+        printZkPathAndTopicString
+      }
+    } else {
+      logger.info(s"Running List Features using zkPath ${params.zkPath}...")
```

Comment:
seems like the if and the else here should have the same logic. It probably makes sense to move the filtering of the topics into the kafka data store.